18-references.Rmd

```diff
@@ -7,12 +7,90 @@
 ```{r include=FALSE}
 # automatically create a bib database for R packages
 knitr::write_bib(c(
-  .packages(), 'bookdown', 'knitr', 'rmarkdown', 'DiagrammeR', 'blogdown', 'kableExtra', 'tinytex', 'stargazer', 'gt', 'flextable', 'magick', 'sass', 'xaringan', 'printr', 'formatR'
-), file = 'packages.bib')
+  .packages(), 
+'knitr', 
+'rmarkdown', 
+'bookdown', 
+'blogdown', 
+'xaringan', 
+'tinytex', 
+'animation', 
+'DT', 
+'tufte', 
+'formatR', 
+'xfun', 
+'pagedown', 
+'remedy', 
+'blastula', 
+'knitcitations', 
+'distill', 
+'gifski', 
+'gganimate', 
+'ggplot2', 
+'DiagrammeR', 
+'nomnoml', 
+'diagram', 
+'dagitty', 
+'ggdag', 
+'govdown', 
+'revealjs', 
+'kableExtra', 
+'flair', 
+'downloadthis',
+'pkgdown',
+'rsconnect',
+'officedown',
+'officer',
+'webshot',
+'flextable',
+'huxtable', 
+'gt',
+'gtsummary',
+'formattable',
+'reactable',
+'rhandsontable',
+'pixiedust',
+'broom',
+'stargazer', 
+'xtable',
+'tables',
+'pander',
+'tangram',
+'ztable',
+'condformat',
+'styler',
+'magick',
+'Cairo',
+'cairoDevice',
+'svglite',
+'tikzDevice', 
+'printr',
+'rgl',
+'reticulate',
+'r2d3',
+'yaml',
+'sass',
+'roxygen2',
+'usethis',
+'rmdformats',
+'flexdashboard',
+'learnr',
+'rticles',
+'spelling',
+'googledrive',
+'workflowr'
```

Comment:
Maybe we could put them into alphabetical order for easier checking in the future ? 

```suggestion
'animation',
'blastula',
'blogdown',
'bookdown',
'broom',
'Cairo',
'cairoDevice',
'condformat',
'dagitty',
'diagram',
'DiagrammeR',
'distill',
'downloadthis',
'DT',
'flair',
'flexdashboard',
'flextable',
'formatR',
'formattable',
'gganimate',
'ggdag',
'ggplot2',
'gifski',
'googledrive',
'govdown',
'gt',
'gtsummary',
'huxtable',
'kableExtra',
'knitcitations',
'knitr',
'learnr',
'magick',
'nomnoml',
'officedown',
'officer',
'pagedown',
'pander',
'pixiedust',
'pkgdown',
'printr',
'r2d3',
'reactable',
'remedy',
'reticulate',
'revealjs',
'rgl',
'rhandsontable',
'rmarkdown',
'rmdformats',
'roxygen2',
'rsconnect',
'rticles',
'sass',
'spelling',
'stargazer',
'styler',
'svglite',
'tables',
'tangram',
'tikzDevice',
'tinytex',
'tufte',
'usethis',
'webshot',
'workflowr',
'xaringan',
'xfun',
'xtable',
'yaml',
'ztable'
```